rails/app/controllers/api/api_controller.rb

```diff
@@ -1,5 +1,7 @@
 class API::APIController < ApplicationController
 
+  rescue_from ActionController::ParameterMissing, with: :parameter_missing
```

Comment:
This seems useful for catching the 5 tests that were failing:

*  3 Failure/Error: name = params.require(:name) ActionController::ParameterMissing: param is missing or the value is empty: name

* 2 Failure/Error: class_word = params.require(:class_word) ActionController::ParameterMissing: param is missing or the value is empty: class_word

But it might have caused some new failures?  122 routing errors are happening now. It might be worth investigating those more closely.